BuildToolsCliVersion.txt

```diff
@@ -0,0 +1 @@
+1.0.0-preview2-002733
```

Comment:
Is BuildTools using a different CLI version than the rest of the build? Will that not cause conflicts? For example, the version above does not include build3 command. Where will we use this?